# HG changeset patch
# Parent 74f6d87cd471e7910e333f927f672d041c857af6
Issue #15822: Prevent spurious pickle re-creation attempts in installed
lib2to3 directory.

diff --git a/Makefile.pre.in b/Makefile.pre.in
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -1079,7 +1079,8 @@
 				fi;; \
 			esac; \
 		done; \
-	done
+	done; \
+	touch $(DESTDIR)$(LIBDEST)/lib2to3/*Grammar*.pickle || true
 	$(INSTALL_DATA) $(srcdir)/LICENSE $(DESTDIR)$(LIBDEST)/LICENSE.txt
 	if test -d $(DESTDIR)$(LIBDEST)/distutils/tests; then \
 		$(INSTALL_DATA) $(srcdir)/Modules/xxmodule.c \
